amxmodx/amxmodx.cpp

```diff
@@ -792,6 +792,11 @@ static cell AMX_NATIVE_CALL is_user_alive(AMX *amx, cell *params) /* 1 param */
 
 	CPlayer* pPlayer = GET_PLAYER_POINTER_I(index);
 
+	if (!pPlayer->initialized)
```

Comment:
can't we just replace this with **pPlayer->ingame** check and remove the other one ?